### DISCUSSION TITLE
Welcome to vidur Discussions!

### DISCUSSION THREAD

**Discussion by amandesai01** [profilecity/vidur#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VenkyGitRep** [comment]
Hi! I'm Venky. I agreed to volunteer for this project as a way to keep my dev skills fresh, while pursuing full time roles.
I'm wondering if there's a slack channel for Vidur? how/where do we talk about issues or ask any questions.

**Reply by amandesai01** [reply]
Hi Venky, welcome to the community! We prefer Discord for discussion. You can find link in readme of the project. Thanks.
